.github/workflows/ci.yml

```diff
@@ -42,8 +42,12 @@ jobs:
       - name: Set up Docker
         run: |
           brew install docker docker-machine docker-compose
+
+          sudo mkdir -p /usr/local/bin 
+          sudo curl -L -o /usr/local/bin/colima https://github.com/abiosoft/colima/releases/download/v0.6.7/colima-Darwin-x86_64 && sudo chmod +x /usr/local/bin/colima
+          brew install lima
```

Comment:
what is this used for?

Reply:
Colima utilizes Lima to create a Linux VM on macOS, where containers can run seamlessly.